src/main/webapp/populateData.js

```diff
@@ -311,3 +312,60 @@ function populateGo() {
       });
 }
 
+/**
+ * Set up the assign panel. For now, this just prints the response
+ * from the server for /gmail-actionable-emails
+ */
+function setUpAssign() {
+  const assignContent = document.querySelector('#assign');
+
+  const subjectLinePhrases = ['Action Required', 'Action Requested'];
+  fetchActionableEmails(subjectLinePhrases, true, 7)
```

Comment:
Perhaps define constants for the other parameters here, so that it's easier to understand which emails are being fetched.

Reply:
Good idea - thanks!